c7n/resources/opsworks.py

```diff
@@ -57,6 +60,54 @@ class resource_type(object):
         date = 'CreatedAt'
         dimension = "StackId"
 
+    retry = staticmethod(get_retry(('ThrottlingException',)))
+    permissions = ('opsworks:ListTags',)
+
+    def augment(self, stacks):
+        filter(None, OpsworkStack.opsworks_tags(
+            stacks, self.session_factory, self.executor_factory, self.retry
+        ))
+        return stacks
+
+    def opsworks_tags(stacks, session_factory, executor_factory, retry):
```

Comment:
since this is now inside the class it should be changed to a non-static method with self as the first parameter, that way you can get rid of the additional args session_factory, executor_factory, and retry. Also, tests are failing in the CI, you can check those before waiting for additional feedback